onair/src/run_scripts/execution_engine.py

```diff
@@ -106,14 +106,31 @@ def parse_configs(self, config_filepath):
         except:
             pass
 
+    def parse_plugins_dict(self, config_plugin_dict): 
+        ## Parse Required Data: Plugin name to path dict
+        ast_plugin_dict = self.ast_parse_eval(config_plugin_dict)
+        if isinstance(ast_plugin_dict.body, ast.Dict):
+            temp_plugin_dict = ast.literal_eval(config_plugin_dict)
+        else:
+            raise ValueError(f"Plugin dict {config_plugin_dict} from {self.config_filepath} is invalid. It must be a dict.")
+
+        for plugin_file in temp_plugin_dict.values():
+            if not(os.path.exists(plugin_file)):
+                raise FileNotFoundError(f"In config file '{self.config_filepath}' Plugin path '{plugin_file}' does not exist.")
+        return temp_plugin_dict
+
     def parse_data(self, parser_file_name, data_file_name, metadata_file_name, subsystems_breakdown=False):
         data_source_spec = importlib.util.spec_from_file_location('data_source', parser_file_name)
         data_source_module = importlib.util.module_from_spec(data_source_spec)
         data_source_spec.loader.exec_module(data_source_module)
         self.simDataParser = data_source_module.DataSource(data_file_name, metadata_file_name, subsystems_breakdown)
```

Comment:
Maybe not part of this merge... but "simDataParser" should be renamed.

Reply:
Understood, should we put in an issue to that effect?

Reply:
https://github.com/nasa/OnAIR/issues/65